src/schema/v1/partner.ts

```diff
@@ -290,14 +290,15 @@ const Partner: GraphQLFieldConfig<void, ResolverContext> = {
       description: "The slug or ID of the Partner",
     },
   },
-  resolve: (_root, { id }, { partnerLoader }, { fieldNodes }) => {
-    const blacklistedFields = ["analytics"]
+  resolve: (_root, { id }, { partnerLoader }, info) => {
+    // when the query is partner analytics, vortex stitching fragment only has _id
+    const blacklistedFields = ["_id"]
```

Comment:
I originally thought we had to blacklist `analytics` for queries like this:
```graphql
query {
  partner(id: "4d8b92c44eb68a1b2c0004cb") {
    analytics {
      topShows: rankedStats(first: 1, objectType: SHOW, period: ONE_YEAR) {
         ...
      }
    }
  }
}
```

but realized when the analytics query gets to this point the query is the [stitching fragment](https://github.com/artsy/metaphysics/blob/master/src/lib/stitching/vortex/stitching.ts#L239-L241) so need to blacklist (whitelist) `_id`